src/plugins/data/config.ts

```diff
@@ -30,3 +30,62 @@ export const configSchema = schema.object({
 });
 
 export type ConfigSchema = TypeOf<typeof configSchema>;
+
+export const searchSessionsConfigSchema = schema.object({
```

Comment:
Had to move the schema now to `src/` because `ese` strategy uses `defaultExpiration` from it.
This schema is still registered by `x-pack/` code